### DISCUSSION TITLE
Disabling HTTP/3 (with QUIC) not working

### DISCUSSION THREAD

**Discussion by psychedelicu** [ZoeyVid/NPMplus#674] [Q&A] [answered]
Hi,

Disabling HTTP/3 (with QUIC) in NPM Plus is not working. I'm using https://http3check.net/ to test the page. 

To work, i have to add this in Advanced > Custom Nginx Configuration:
http3 off;
http3_hq off;

Maybe a bug?

edit:
I discovered that if i restart NPM Plus, the HTTP/3 is on again. I have to edit the proxy host and save it to work again v2. 

2024/03/03 14:00:05 [warn] 766#766: *424 a client request body is buffered to a temporary file /usr/local/nginx/client_body_temp/0000000028, client: 86.121.186.118, server: drive.domain.com, request: "PUT /remote.php/dav/uploads/psy/web-file-upload-4f498e44910766e2/9 **HTTP/3.0**", referrer: "https://drive.withpdomainy.com/apps/files/files/1445?dir=/Test"

**Comment by Zoey2936** [comment, marked as answer]
will be fixed in the next release
